compiler/damlc/daml-stdlib-src/DA/Internal/Exception.daml

```diff
@@ -77,45 +79,57 @@ instance HasFromAnyException AnyException where
 
 --------------------------------------------------------------
 
+instance HasField "message" GeneralError Text where
+    getField (GeneralError m) = m
+    setField m (GeneralError _) = GeneralError m
+
 instance HasThrow GeneralError where
     throwPure = primitive @"EThrow"
 
 instance HasMessage GeneralError where
-    message = primitive @"BEGeneralErrorMessage"
+    message (GeneralError m) = m
 
 instance HasToAnyException GeneralError where
     toAnyException = primitive @"EToAnyException"
 
 instance HasFromAnyException GeneralError where
-    fromAnyException = primitive @"EFromAnyException"
+    fromAnyException = handleBuiltinException "GeneralError" GeneralError (primitive @"EFromAnyException")
```

Comment:
Will we ever throw this? I thought we agreed that `error` is just going to bypass the LF primitive and throw the LF stdlib primitive.

Reply:
This is for catching, not throwing.

Reply:
Right but if we throw `GeneralError` directly in Daml-Stdlib and LF never throws this then we don’t need to go via `Text` here.

Reply:
Ah, true! I guess the question is do we ever want to throw a GeneralError in LF?

Reply:
my understanding is we don’t but we don’t have to solve this in this PR. Maybe just add a todo.